src/EnergyPlus/UnitarySystem.cc

```diff
@@ -9758,7 +9766,7 @@ namespace UnitarySystems {
                 int ATMixOutNode = this->ATMixerOutNode;
                 SensOutput =
                     DataLoopNode::Node(ATMixOutNode).MassFlowRate * (Psychrometrics::PsyHFnTdbW(DataLoopNode::Node(ATMixOutNode).Temp, MinHumRatio) -
-                                                                     Psychrometrics::PsyHFnTdbW(RefTemp, MinHumRatio));
+                                                                     Psychrometrics::PsyHFnTdbW(RefTemp, MinHumRatio) - this->m_SenLoadLoss);
```

Comment:
noticed this when playing with SensOutput calculation (i.e., using new Bereket psyc function). Left as is to not muddy the water but noticed that this variable was missing.

Reply:
@lgu1234 do these sensible capacity calculations need to include the SensLoadLoss variable?

Reply:
@rraustad The SensLoadLoss is duct loss for a whole distribution system. If this system has a single terminal, it needs to add it. However, if there are more terminals or more zones are served, the SensLoadLoss should be added. However, the loss should be added weighted by mass flow rates.